scripts/src/public/services/payment.ts

```diff
@@ -123,7 +144,7 @@ export async function getBlockchainConfig(blockchainVersion: BlockchainVersion):
 	return res.data;
 }
 
-function getPaymentServiceUrl(blockchainVersion: BlockchainVersion): string {
+export function getPaymentServiceUrl(blockchainVersion: BlockchainVersion): string {
```

Comment:
I think after deleting the whitelistTransaction function you can remove this `export` and leave it internal